eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -268,6 +287,19 @@ instance
   ) =>
   ConwayEraImp (ConwayEra c)
 
+registerInitialCommittee ::
+  forall era.
+  ConwayEraImp era =>
+  ImpTestM era (NonEmpty (Credential 'HotCommitteeRole (EraCrypto era)))
+registerInitialCommittee = do
+  hk <- registerCommitteeHotKey initialCommitteeCred
```

Comment:
Global bindings like that is a bad idea. Look up the committee cold credential in the state instead